src/features/profile/Profile.tsx

```diff
@@ -22,40 +30,104 @@ interface ProfileProps {
   navigation: NavigationProp;
 }
 
-const profile = ({navigation}: ProfileProps) => {
-  const logOutHandler = () =>
+interface Props extends ProfileProps {
+  logOutHandler: () => void;
+  setTheme: (theme: string) => void;
+  theme: string;
+}
+
+const profile: React.FC<Props> = ({
+  navigation,
+  logOutHandler,
+  setTheme,
+  theme,
+}) => {
+  const logoutHandler = () => {
+    logOutHandler();
     navigation.reset({routes: [{name: Routes.Login}]});
+  };
 
+  const {colors} = useTheme();
+  const [checked, setChecked] = useState('light');
   return (
-    <View style={profileStyles.profileScreen}>
-      <View style={profileStyles.profileContainer}>
+    <View
+      style={[profileStyles.profileScreen, {backgroundColor: colors.primary}]}>
+      <View
+        style={[
+          profileStyles.profileContainer,
+          {backgroundColor: colors.card},
+        ]}>
         <Image
           style={profileStyles.profilePicture}
           source={{uri: 'https://picsum.photos/200'}}
         />
         <View style={profileStyles.textContainer}>
-          <Text style={profileStyles.profileText}>somename</Text>
-          <Text style={profileStyles.profileText}>someemail</Text>
+          <Text style={[profileStyles.text, {color: colors.text}]}>
+            John Smith
+          </Text>
+          <Text style={[profileStyles.text, {color: colors.text}]}>
+            {testData.email}
+          </Text>
+        </View>
+      </View>
+      <View style={profileStyles.themeChangerContainer}>
+        <View style={profileStyles.radioContainer}>
+          <Text style={[profileStyles.text, {color: colors.text}]}>
+            Change theme to light
+          </Text>
+          <RadioButton
+            style={
+              checked === 'light'
```

Comment:
this styles should be inside radiobutton we dont handle it outside of radiobutton component ...